GitUp/Application/AppDelegate.m

```diff
@@ -550,17 +551,9 @@ - (IBAction)viewIssues:(id)sender {
 }
 
 - (IBAction)showAboutPanel:(id)sender {
-#if DEBUG
-  _versionTextField.stringValue = @"DEBUG";
-#else
-  if (_updatePending) {
-    _versionTextField.stringValue = NSLocalizedString(@"Update Pending", nil);
-  } else {
-    _versionTextField.stringValue = [NSString stringWithFormat:NSLocalizedString(@"Version %@ (%@)", nil), [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleShortVersionString"], [[NSBundle mainBundle] objectForInfoDictionaryKey:@"CFBundleVersion"]];
-  }
-#endif
-  _copyrightTextField.stringValue = [[NSBundle mainBundle] objectForInfoDictionaryKey:@"NSHumanReadableCopyright"];
-  [_aboutPanel makeKeyAndOrderFront:nil];
+  AboutPanelWindowController *aboutPanel = [[AboutPanelWindowController alloc] init];
+  [aboutPanel populateWithDataWhenUpdateIsPending:_updatePending];
+  [aboutPanel showWindow:nil];
```

Comment:
Actually, I forgot that we only ever want one instance of this window, so we'll want to make sure we're loading this windowController only once. I'd personally prefer lazy loading via property. Something like:
```
- (AboutPanelWindowController *)aboutPanelWindowController {
    if (!_aboutPanelWindowController) {
        _aboutPanelWindowController = [[AboutPanelWindowController alloc] init];
    }
    return _aboutPanelWindowController;
}
```
Then simply reference `[self.aboutPanelWindowController showWindow:nil]` to have it always be a single instance.

Reply:
Also, Panel and Window are redundant. I'd say call it either `AboutWindowController` or `AboutPanelController`.